platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -222,7 +305,7 @@ def get_presence(self):
         return eeprom_raw is not None
 
     # Read out any bytes from any offset
-    def read_eeprom(self, offset, num_bytes):
+    def _read_eeprom_specific_bytes(self, offset, num_bytes):
```

Comment:
As we discussed **_read_eeprom_specific_bytes** is not used anymore. All code that used it is removed.
New implementation of **read_eeprom** should return the same data as ethtool returns